README.md

```diff
@@ -54,6 +54,12 @@ To restore MySQL, MongoDB, and Caddy from a previously-made backup tar file:
 This will look for the tar file under `$(tutor config printroot)/env/backup/`,
 extracts the tar file, and restore the services from the backup files.
 
+You can also exclude specific data from the restore. For example, if
+you want to restore only MySQL data, and leave the state of MongoDB
+and Caddy untouched, run:
+
+    tutor k8s restore --exclude=caddy --exclude=mongodb
```

Comment:
We haven't done this yet actually. We have only added `--exclude` to the Python scripts that run inside the container.

Reply:
I'll create a PR for adding `exclude` to the restore commands.

Reply:
#15 will do the trick. We should merge that and rebase this PR.